src/common/authRoleStatus.component.tsx

```diff
@@ -0,0 +1,65 @@
+import { IconButton, Paper, Tooltip, Typography } from '@mui/material';
+import Grid from '@mui/material/Grid2';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import { getUserRole } from '../parseTokens';
+
+function AuthRoleStatus() {
+  const role = getUserRole();
+
+  return (
+    role && (
+      <Paper
+        square
+        sx={{
+          maxHeight: '40px',
+          backgroundColor: '#FFA500',
+          display: 'flex',
+          paddingLeft: 0,
+          paddingRight: 5,
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Grid
+          container
+          direction="row"
+          alignItems="center"
+          justifyContent="center"
+          aria-label="admin-status-info"
+        >
+          <Grid>
+            <Tooltip
+              title={
+                'As a privileged user you can create/delete units and usage statuses, and bypass rules when creating, deleting, editing, or moving items.'
```

Comment:
The text should use "Admin user" rather than "Privileged user". Similar to above, you need to mention that this is on top of the standard features